.github/workflows/check_file_size.yml

```diff
@@ -0,0 +1,43 @@
+name: Check image sizes
+
+on:
+  pull_request:
+    paths:
+      - "**.jpg"
+      - "**.jpeg"
+      - "**.png"
+      - "**.webp"
+
+jobs:
+  check-files-size:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    concurrency:
```

Comment:
What's the reason we need the `concurrency` configuration?

Reply:
This is for the `cancel-in-progress` feature, but we can remove it because this workflow is fast, it's just simple bash script